book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals_list
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals_list = []
+  end
+
+  def add_rental(rental)
+    @rentals_list.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] You already pushed the rental into the rentals_list of the book in the Rental class:
```rb
  def initialize(date, person, book)
    @date = date
    @person = person
    @book = book
    book.rentals_list.push(self) # here
    person.rentals_list.push(self)
  end
```
So in the add_rental method, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association:
```suggestion
  def add_rental(rental)
     Rental.new(date, self, person)
  end
```